src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -261,11 +261,14 @@ impl ThinPool {
         INITIAL_MDV_SIZE
     }
 
+    pub fn status(&mut self, dm: &DM) -> EngineResult<dm::ThinPoolStatus> {
```

Comment:
I don't think this method is needed for testing.

Reply:
Why not?  I'm using it to verify the meta total allocated space is increased.  Isn't it needed to validate the meta was expanded?

Reply:
The only test that you add is in this file, so you could just as well use ```thin_pool.thin_pool.status(&dm)``` in your test. There is no need for making anything public.

Reply:
I see.  thanks.